src/services/bookMark.service.ts

```diff
@@ -0,0 +1,44 @@
+import { PrismaClient } from '@prisma/client';
+import { Service } from 'typedi';
+import { CreateBookmarkDto } from '@/dtos/bookMark.dto';
+import { HttpException } from '@exceptions/HttpException';
+import { Bookmark } from '@/interfaces/bookMark.interface';
+
+@Service()
+export class BookmarkService {
+  public bookmarks = new PrismaClient().bookmark;
+
+  public async createBookmark(bookmarkData: CreateBookmarkDto): Promise<Bookmark> {
+    const { userId, housePostId } = bookmarkData;
+
+    const createBookmarkData: Promise<Bookmark> = this.bookmarks.create({
+      data: {
+        user: {
+          connect: { id: userId }
+        },
+        housePost: {
+          connect: { id: housePostId }
+        }
+      }
+    });
+    return createBookmarkData;
+  }
+
+  public async deleteBookmark(userId: number, housePostId: number): Promise<Bookmark> {
```

Comment:
why are you deleting like this? why not with bookMarkId? Also returning a bookmark is wierd

Reply:
good point, i can do that. I went like this because of some FE decisions I made. but when I think about it now, I can use actual ids.